### ISSUE TITLE
hw04 ready for grading

### ISSUE THREAD

**Issue by k3vzhu** [k3vzhu/STAT545-hw-Zhu-Kevin#5]
[hw05 README](https://github.com/k3vzhu/STAT545-hw-Zhu-Kevin/blob/master/STAT545-hw05-Zhu-Kevin/STAT545-hw05-Zhu-Kevin-README.md)  
[hw05 md](https://github.com/k3vzhu/STAT545-hw-Zhu-Kevin/blob/master/STAT545-hw05-Zhu-Kevin/STAT545-hw05-Zhu-Kevin.md)

**Comment by aiod01** [comment]
Hello, I am Hyeongcheol Park. Thank you for your mail and inviting me on your GitHub!
To sum up what I have learned from your work and what I was especially impressed;

1. It is very nice to make links to every content. I wish I could have done that in my assignment!

2. Before you do coding, you explained what you will do on sentence. It was kind of you to help me follow the process very well.

3. The idea of comparing trimmed mean with plain mean, adding which function,  is interesting and brilliant idea I think!

4.  You save variable in column level such as dat_3$country<-dat_2$country. Good to know that we can do this.

5. thanks to let me know that when we create csv file there appear index variable X.
 

6. One more good to know thing: fct_other seems nice to use when we compare things in a binary way!


If I should give you some suggestion for your homework even though I actually do not find any flaw, I may say that After print(plot4b), you may use <br/> to separate the sentence. But overall, It was just a great work. And I think your GitHub and all .md files are so well organized!

To sum up, it was just nice work. I enjoyed exploring your GitHub. Thank you and I hope you enjoyed learning!

**Comment by sdhanani** [comment]
Good job dropping Oceania from the data removing it as a factor from the dataset. You chose a unique way to reorder the levels of countries in the dataset and show that clearly. You also compare two different types of reading and writing files into and out of R and included descriptions of whether or not the formatting was saved throughout the process. Your graph improved as you went through the steps of visualization but the y-axis is a little difficult to read. You use ggsave() but it would have been nice to see you embed the plot into your RMarkdown file. I really like your idea for the "I want to do more" section and you create the new factor by animal well. Your repo has a few extra files that could have been put into one of the homework folders instead of the main page of your repo and you have a table of contents with links to each assignment but it would have been handy to have a small description of what each assignment contains for future you. Overall, well done.

**Comment by pgonzaleze** [comment]
Hi @k3vzhu here are some comments about your homework:
Reorder a factor: yes 
File I/O: yes
Implement visualization: yes
Write data to file: yes
Clean up: yes
Extras: yes
#### Part I
+ Nice approach,
+ I liked the brief explanation that you gave before and after the outputs in both activities of this task, 
+ The code is nice and clear, clearly you are getting familiar with R,
#### Part II
+ I think I gave this advice before but I try to not use simple names as dat_1...2…etc, just because at some point could create confusion, any way the code is good and the task as well so if you feel comfortable doing/writing in this way go for!
+ Nice exploration, also good to pointed out that the “x” column appears/disappears,
#### Part III
+ Nice example(s) to tackle the question,
+ I liked that you gave the explanation before and after, in this case it was easy to check and compare the figure(s), 
+ Great integration of functions in your code,
#### Part IV
+ Well use of ggsave()
#### Part V
+ Tidy repo and well organised.
#### Part VI
+ Nice approach, it might be useful to see an extract of the table with the “Asia and other” continents working.
+ Good example to add a factor, 

Overall, great homework!
Your grades would be distributed later,
Regards, 

Pedro G.
